lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -479,6 +455,22 @@ class JsonpMainTemplatePlugin {
 				return source;
 			}
 		);
+		mainTemplate.hooks.beforeStartup.tap(
```

Comment:
Trigger prefetching after the startup code runs. This would omit prefetching when `import()` is used in initialization code.

Reply:
Where? There's no `afterStartup` hook, and `startup` includes a return statement that prevents later code from executing.